tests/nnstreamer_converter/unittest_converter.cc

```diff
@@ -184,6 +185,124 @@ TEST (tensorConverterCustom, invalidParam3_n)
   EXPECT_NE (0, nnstreamer_converter_custom_unregister ("tconv"));
 }
 
+/** @brief tensor converter plugin's query caps callback */
+static GstCaps *
+conv_query_caps (const GstTensorsConfig *config)
+{
+  return NULL;
+}
+
+/** @brief tensor converter plugin's get out caps callback */
+static gboolean
+conv_get_out_config (const GstCaps *in_cap, GstTensorsConfig *config)
+{
+  return TRUE;
+}
+
+/** @brief tensor converter plugin's convert callback
+ */
+static GstBuffer *
+conv_convert (GstBuffer *in_buf, gsize *frame_size, guint *frames_in, GstTensorsConfig *config)
+{
+  return NULL;
+}
+
+/**
+ * @brief Get default external converter
+ */
+static NNStreamerExternalConverter *
+get_default_external_converter (const gchar *name)
+{
+  NNStreamerExternalConverter *sub = g_try_new0 (NNStreamerExternalConverter, 1);
+  g_assert (sub);
+
+  sub->name = (char *) name;
+  sub->query_caps = conv_query_caps;
+  sub->get_out_config = conv_get_out_config;
+  sub->convert = conv_convert;
+
+  return sub;
+}
+
+/**
+ * @brief Test for plugin registration
+ */
+TEST (tensorConverter, subpluginNoraml)
+{
+  NNStreamerExternalConverter *sub = get_default_external_converter ("mode");
+  EXPECT_TRUE (registerExternalConverter (sub));
+  g_free (sub);

```

Comment:
Add code to unregister ext-converter.
OR
```
remove_default_external_converter (const gchar *name)
{
  var = _conv_find (name);
  if var
    removeextconverter (name);
}
```

Reply:
Updated!